src/components/SoraCard/steps/ToSDialog.vue

```diff
@@ -19,12 +30,65 @@ import TranslationMixin from '@/components/mixins/TranslationMixin';
 })
 export default class TermsAndConditionsDialog extends Mixins(TranslationMixin, mixins.DialogMixin) {
   @Prop({ default: '', type: String }) readonly srcLink!: string;
+  @Prop({ default: '', type: String }) readonly title!: string;
 
   loading = true;
+  flags = [];
+
+  unsupportedCountriesCodes = [
```

Comment:
```
unsupportedCountries = {
  us: 'USA', // please avoid UPPER CASE
  ...
  ru: 'Russia'
}
```

this.flags = Object.keys(this.unsupportedCountries).map(countryCodeEmoji); // `toUpperCase` if needed